tests/integration/test_charm.py

```diff
@@ -171,3 +174,52 @@ async def test_invalid_jvm_config(ops_test: OpsTest) -> None:
                 "Invalid jvm-config, "
                 "Java heap memory specification exceeds application memory constraint"
             )
+    await asyncio.gather(
+        *(
+            [
+                ops_test.model.applications[app_name].set_config({"jvm-config": ""})
+                for app_name in ALL_APP_NAMES
+            ]
+            + [ops_test.model.wait_for_idle(apps=ALL_APP_NAMES)]
+        )
+    )
+
+
+async def test_nginx_ingress(ops_test: OpsTest) -> None:
```

Comment:
Rename to `test_ingress` to have a generic test

Reply:
Updated, thanks!